app/Nova/Resource.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova;
+
+use Illuminate\Database\Eloquent\Builder;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Laravel\Nova\Resource as NovaResource;
+
+abstract class Resource extends NovaResource
+{
+    public static ?array $defaultSort = null;
```

Comment:
I think it's safer to make this protected. No need to set it from outside a Resource.
But I think `indexQuery` needs to stay public if I see how this works inside the Laravel Nova code?

Reply:
Good find, I'll make it protected.